package.json

```diff
@@ -58,6 +58,7 @@
     "concurrently": "^3.4.0",
     "cpx": "^1.5.0",
     "mocha": "^5.2.0",
+    "office-addin-manifest": "^1.2.4",
```

Comment:
This should not be in both dependencies and devDependencies.

Reply:
fixed